otx/algorithms/action/tasks/inference.py

```diff
@@ -307,7 +307,7 @@ def unload(self):
             self._delete_scratch_space()
 
     @check_input_parameters_type()
-    def export(self, export_type: ExportType, output_model: ModelEntity):
```

Comment:
Why do we need kwargs here? They are not used

Reply:
Yes, we don't use kwargs (or dump_features) in action and anomaly tasks, but since I changed the [line in cli/export](https://github.com/openvinotoolkit/training_extensions/blob/5de193dd4080117a2e850e56345de3b1cd95a242/otx/cli/tools/export.py#L92) and added this parameter to parent IExport class, I need to have it in child classes as well, even if I don't use it.

It's an inconsistent behavior, that will be fixed after the release

Reply:
In that case, we shouldn't break the interface conformance and make the method signature unclear.
My proposal is to follow the interface, but raise a NotImplementedError if dump_features is false.

Reply:
It's a great suggestion to keep the interface and code clear, I was wondering how to achieve it. Thank you for this idea.